reposilite-frontend/src/App.vue

```diff
@@ -15,33 +15,28 @@
   -->
 
 <template>
-    <div id="app">
-        <Welcome v-if="isWelcome()"/>
+    <div id="app" :class='{ welcome }'>
+        <Welcome v-if="welcome"/>
         <Dashboard v-else/>
     </div>
 </template>
 
 <script>
-import Menu from './components/Menu.vue'
 import Welcome from './components/Welcome'
 import Dashboard from './components/Dashboard'
 
 export default {
     components: {
         Dashboard,
-        Welcome,
-        Menu
+        Welcome
     },
-    methods: {
-        getUser() {
+    computed: {
+        user: function () {
             return this.$store.state.user
         },
```

Comment:
Add 
```js
import { mapState } from 'vuex'
```
and then change this to
```js
...mapState({
  user: state => state.user
}),
```

It's better for debugging with Vue Devtools